async-http-client-backend/fs2/src/main/scala/sttp/client3/asynchttpclient/fs2/AsyncHttpClientFs2Backend.scala

```diff
@@ -119,46 +119,41 @@ object AsyncHttpClientFs2Backend {
         apply[F](
           AsyncHttpClientBackend.defaultClient(options),
           closeClient = true,
-          blocker,
           customizeRequest,
           webSocketBufferCapacity
         )
       )
 
   /** Makes sure the backend is closed after usage.
     */
-  def resource[F[_]: ConcurrentEffect: ContextShift](
-      blocker: Blocker,
+  def resource[F[_]: Async: Dispatcher](
```

Comment:
this could remove the `Dispatcher` constraint, and you build one in this method (which returns `Resource`)

Reply:
https://github.com/softwaremill/sttp/pull/738/commits/e93160fc787f6ddee9797349642bc4bad94539d7